tests/test_cachito_config.py

```diff
@@ -54,11 +54,13 @@ def test_validate_cachito_config_cli(mock_isdir, app):
         ([["gomod"]], False),
     ],
 )
-def test_validate_mutually_exclusive_package_managers(app, value, is_valid):
+def test_validate_mutually_exclusive_package_managers(app, value, is_valid, tmpdir):
     config = app.config.copy()
     config["CACHITO_MUTUALLY_EXCLUSIVE_PACKAGE_MANAGERS"] = value
 
     if is_valid:
+        # Successful validation requires this dir exists.
+        config["CACHITO_BUNDLES_DIR"] = str(tmpdir)
```

Comment:
It looks like the other tests handle this by mocking:
```python
@patch("os.path.isdir", return_value=True)
```

Reply:
Yeah, I tend to make it explicit for this. I actually also think it should be better ensure the directory exists than mocking the isdir :) IMO, it makes it much clear what the test tests.